Wishlist/wishlist.css

```diff
@@ -0,0 +1,31 @@
+@import url("/style.css");
+.card-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+}
+
+.cards {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
+    justify-content: space-around;
+}
+
+.action-btns {
+    display: flex;
```

Comment:
Suggestion : this could be a seperate utility class.

Reply:
A utility class  is already defined for this use, for some reason, it was not working. So, i rewrote the code here